src/utils.js

```diff
@@ -89,16 +89,11 @@ const getDBFilePathWithConf = (DBFile) => {
 };
 
 const getDBFilePath = (chromeProfilePath, DBFile) => {
-  switch (conf['browser']) {
-  case 'Chrome Canary':
-    return `/Users/${userName}/Library/Application Support/Google/Chrome Canary/${chromeProfilePath}/${DBFile}`;
-  case 'Edge':
-    return `/Users/${userName}/Library/Application Support/Microsoft Edge/${chromeProfilePath}/${DBFile}`;
-  case 'Chromium':
-    // 'Chrome Cloud Enrollment' could be wrong (not sure)
-    return `/Users/${userName}/Library/Application Support/Google/Chrome Cloud Enrollment/${chromeProfilePath}/${DBFile}`;
-  default:
-    return `/Users/${userName}/Library/Application Support/Google/Chrome/${chromeProfilePath}/${DBFile}`;
```

Comment:
We cannot remove the codes here.
When `browser_dir` is not specified, `getDBFilePath` return undefined. (`browser_dir` is optional)

Reply:
Or you could set "browser_dir" in config with a default value of "Chrome Path", which is what I went with as it seemed *more efficient*. We can always set it as a default and still have changing it as optional, much like all the other config items.

Reply:
Then users should know the DB filepath, not just the browser name. (DB filepath here varies slightly from browser to browser.)
This is breaking change and I don't see it as more efficient.
And it doesn't make sense `getDBFilePath` might return `undefined`.